docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
+
+{WFM-RC-NameShort} authentication and authorization module using link:http://passportjs.org[PassportJS].
+This module allows the:
+- Creation and initialization of a Passport authentication service using Passport's local strategy.
```

Comment:
That's too technical in my opinion. We can have it in README but docs needs to introduce things slowly. 
Assume that we do not need to explain how passport works - just why we decided to wrap that into module etc.